core/controller/src/main/scala/org/apache/openwhisk/core/controller/Actions.scala

```diff
@@ -102,6 +104,12 @@ trait WhiskActionsApi extends WhiskCollectionAPI with PostActionActivation with
   /** Database service to get activations. */
   protected val activationStore: ActivationStore
 
+  /** Config flag for Execute Only for Actions in Shared Packages */
+  protected def executeOnly =
+    Try({
+      loadConfigOrThrow[Boolean](ConfigKeys.sharedPackageExecuteOnly)
+    }).getOrElse(false)
```

Comment:
Generally the convention so far is to put all defaults into applicaiton.conf file. So I would a) add a default value (false) for `whisk.shared-packages-execute-only` into application.conf, and b) remove the `Try` here since it should always work.